live-examples/html-examples/embedded-content/source.html

```diff
@@ -0,0 +1,12 @@
+<video controls
+    width="300"
+    height="200"
+    autoplay="true"
+    loop="true"
+    muted="true">
+    <source src="/media/examples/stream_of_water.webm" type="video/webm">
```

Comment:
If we break this line and indent so the attributes are aligned, then it will be more readable at the most likely widths:

```
<source src="/media/examples/stream_of_water.webm"
        type="video/webm">
```